src/config.js

```diff
@@ -194,6 +194,43 @@ const config = {
         isPercent: true
       }]
     }
+  },
+  charts: {
+    histograms: {
+      people: {
+        field: 'den14',
+        title: 'HISTOGRAM_PEOPLE_TITLE'
+      },
+      households: {
+        field: 'hh14',
+        title: 'HISTOGRAM_HOUSEHOLDS_TITLE'
+      },
+      economy: {
+        field: 'ent14',
+        title: 'HISTOGRAM_ECONOMY_TITLE'
+      },
+      infrastructure: {
+        field: 'pcwat10',
+        title: 'HISTOGRAM_INFRASTRUCTURE_TITLE'
+      }
+    },
+    height: 200,
+    width: undefined, // Width is dynamically detected from page state
+    callouts: { // Callouts are the vertical bars representing specific values
+      country: {
```

Comment:
I'm assuming these keys have some meaning? Is it the db column to query for? If so, consider documenting here with a comment for future reference.